lib/buffer.js

```diff
@@ -1277,13 +1279,43 @@ function atob(input) {
       // The first 5 elements of `kForgivingBase64AllowedChars` are
       // ASCII whitespace char codes.
       nonAsciiWhitespaceCharCount++;
+
+      if (index === kForgivingBase64AllowedChars.length - 1) {
+        // The last element of `kForgivingBase64AllowedChars` is the `=`
+        equalCharCount++;
+      }
```

Comment:
```suggestion
      } else if (equalCharCount) {
        // Only one more `=` is permitted after the first equal sign.
        throw lazyDOMException('Invalid character', 'InvalidCharacterError');
      }
```

Reply:
`ab=     =    ` is legal.

Reply:
We're inside ` if (index > 4) {` block, whitespaces are already handled.